.buildkite/pipeline.yml

```diff
@@ -383,17 +383,23 @@ steps:
           slurm_mem: 20GB
 
       - label: ":balloon: Single column EDMF GABLS"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config column --FLOAT_TYPE Float64 --hyperdiff false --moist dry --edmf_coriolis GABLS --turbconv edmf --turbconv_case GABLS --split_ode false --ode_algo ODE.Euler --dt_save_to_sol 5mins --z_elem 8 --z_stretch false --z_max 400 --job_id edmf_gabls --dt 4secs --t_end 9hours --regression_test true --anelastic_dycore true --apply_limiter false --debugging_tc true --dt_save_to_disk 5mins"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config column --FLOAT_TYPE Float64 --hyperdiff false --moist dry --edmf_coriolis GABLS --turbconv edmf --turbconv_case GABLS --split_ode false --ode_algo ODE.Euler --dt_save_to_sol 5mins --z_elem 8 --z_stretch false --z_max 400 --job_id edmf_gabls --dt 500secs --t_end 9hours --regression_test true --anelastic_dycore true --apply_limiter false --debugging_tc true --dt_save_to_disk 5mins"
```

Comment:
Won't this be unstable with explicit Euler?

Reply:
This one is actually even more stable than ARS343; its max dt is between 600 and 700, while the ARS343 max dt is between 500 and 600.

Reply:
See here for my full dt sweep: https://buildkite.com/clima/climaatmos-ci/builds/5998

Reply:
Wait, explicit is more stable than implicit?

Reply:
*anelastic is more stable than compressible

Reply:
Ok, that's not really a fair comparison. Also, I just looked at the plots, and the solution is checker-boarding (it's unstable). Let's leave this job alone for now and focus on the compressible cases.

Reply:
Ok sure. @trontrytel I'm going to leave the anelastic job's dt unchanged.

Reply:
The Newton's method job looks unstable, too.

Reply:
Wait so how exactly should I get maximum dt values then? I was operating under the assumption that `simulation runs ==> stable`

Reply:
Like, for example, is [this](https://s3.amazonaws.com/buildkiteartifacts.com/52b8ca89-71e4-4c6b-b9a2-94f4dcea2227/ce4778fd-42ef-4a79-b4bc-c3dbffbfae4b/0184d518-667a-4ec7-b51e-31fb72ca517e/0184d519-3908-4e22-89fd-9686adf9b56f/compressible_edmf_gabls_jfnk_true_true_700_0.1_1.0/____________________________________final_contours.png?response-content-type=image%2Fpng&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7LVELGW53K%2F20221206%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20221206T002639Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEOD%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJGMEQCIH5Q2%2FCb31olJq4BTDSlqc0g3IqsX7RyLoee1OBUoGmQAiAYK7mEGZRkivc85e6deWZbhDu1%2FhvXKXgTJ4Ak7xASKirVBAj5%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F8BEAAaDDAzMjM3OTcwNTMwMyIMPPX2o2qgBRD6a54BKqkEIQ0A0cLxKGuk5Zv%2FvS4%2FTLGmhpXJ6lH0hfUDEsKDoJxhJptFIbFlxN3iN3PcRrSyEu9q6jvHyLBXr1uh7IVhe%2FupYWd7Xi%2FVS8rkQAUH4mGPmV0GbBwKB7Pvx0DXV7yGPjSsy94e9o9JQUxwmMl7Py5pNhiY0aPeCZvHE8U3JEubC7y65kb%2FzuDHQDZp6rpAM1bkMnUXzNP2hcgauAf7IaAxvfLvtvt0hknN2yUpmp0utbCfvPo7jSQnJRk1mZTwruY%2B%2FvGA9JxGZJOQ1lSG0cW7J6fOkL7qUjk%2F9AVWiC4Y%2Ft1Ir0KkAlVIydgNhovMRxsvge6GTaxPJFjAuLrM5UkP8n8OB9l5vljxJYBI0nVQPd%2BpjNMkQfTnLa5OIc79b6va7EzpLd%2Bb3jlX0NZq3a1xahZyFQUTe2OljBuMqTUGywNmHR4Y3K%2BdoF9XBBo5lACD71Vt2fWPxcnv0OWrTs7QRm10GIKZDJhUOE8ksTZbVo50ZH9pc0EqBprDn5E0uIvJgzxS4iKZoQfTDljkSGhplUZ3ADyBb%2FQ7PXztIveKh4DhFili%2BHFB%2BcQnyD6gnSup%2B72erUiLe9O0jCTjrnWKRyM5wF4S7XAYuzobwTl9zhWnPmTqE7%2BjzMmJN0uZP%2Fs7LApAik8hpi1j4TQGzGGixGSuFifMZVX00fs35FnzaUVYKAPKwowldfFVRVDPxOlMA%2BJdg3k8QL7o2lEjz4yKtp%2BDELiEZDDkibqcBjqqAdlSUJl2oN6NTERyjx4ctnsgL%2FC8ar%2FdZnUAabrjI%2BYT3HuPGKmIR0mGuBa0Tjts0BVioSDHT3FA%2FTizgB5FuIC76%2FsKdIHXVTtVu3fRBWFccmuTLAqT%2FuY4B39oXP3WqSgZDuUbxrsoCynSDyLEP7xGPcMZsmZXgrtDqbvDFx4NHPjPWlzXIhgbJGd1vFOHaYDhvPin76bjXaCYIb3fThSB%2BsSjY2faK1K%2F&X-Amz-SignedHeaders=host&X-Amz-Signature=d34fea5c9cf1f309943f41e92e61fec81cf8c0c72b2f57ef2d813395b1ec5bd4) (JFNK with IMEX EDMF and dt = 700) stable?

Reply:
> Wait so how exactly should I get maximum dt values then?

You need to look at the _quality_ of the results

> I was operating under the assumption that `simulation runs ==> stable`

I think this assumption is perhaps technically correct, but we care about the quality of the results, too. I think the right way to think about this is: if we were to stably solve this explicitly, the results would have a certain "quality", and this is the quality that we should be comparing with when solving implicitly. After all, implicit timestepping is an optimization, we shouldn't be optimizing by increasing `dt` above which our solution quality degrades (which is clearly happening in this PRs)-- it's no longer a fair comparison because each approach is better in their own way.

Reply:
Agree 100%. It's good to know for what dts our simulations still "don't crash". But we should keep dt in the CI such that the solutions are good quality

Reply:
Seems to me like the dt for EDMF GABLS Newtons method is ok but for JFNK with IMEX we should probably reduce it a little? I think our target dt is ~100s anyhow